.github/workflows/glean.yaml

```diff
@@ -0,0 +1,25 @@
+name: GLEAN probe-scraper
+
+on:
+  workflow_call:
```

Comment:
This line seems strange. Is this defining an empty map?

Reply:
This is directly copied from [the reusable workflow docs](https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow), and is equivalent to:

```
on:
  workflow_call: null
```

we don't need any inputs or secrets, but if we did it would be like this:

```
on:
  workflow_call:
    inputs:
      username:
        required: true
        type: string
    secrets:
      envPAT:
        required: true
```